### DISCUSSION TITLE
QucsConverter: should it work on datasets?

### DISCUSSION THREAD

**Discussion by Rmano** [ra3xdh/qucs_s#742] [Q&A] [answered]
Hi all!

Is `qucsconverter` supposed to work? I see that this is a **very** old piece of software... I tried to use it on one simulation output dataset:

```
> qucsconv -if qucsdata -i feedback_par_par.dat.ngspice  -of csv
line 2: syntax error, unrecognized character: `('
line 2: syntax error
```

The file starts as

```
<Qucs Dataset 24.2.1>
<indep i(vic1) 1>
1.054130000000e-03+j0.000000000000e+00
[...]
```

I was trying to find the source code to have a look at it but I am unable to find it... 😳

**Comment by ra3xdh** [comment, marked as answer]
Qucsconv is now a part of Qucs-S distribution as `qucsconv_rf`. See the sources here: https://github.com/ra3xdh/qucsator_rf in the `src/converter` subdirectory. See #370 and #647 It seems the qucsonv doesn't accept the X-axis variable using SPICE syntax. I have created the dedicated issue https://github.com/ra3xdh/qucsator_rf/issues/4

**Comment by ra3xdh** [comment]
The SPICE variables support for Qucsconv has been added by https://github.com/ra3xdh/qucsator_rf/pull/8
